### PR TITLE
Update CHANGELOG.md with v3.8.1 changes

### PR DESCRIPTION
**Pull request by dataclouder**
Update `CHANGELOG.md`

### CODE REVIEW

**Review by Didainius — APPROVED** [review]

**Review by adambarreiro — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
+* Add `vdc_id` to data source `vcd_vm_placement_policy` to allow tenant users to fetch VM Placement Policies from
+  the ones assigned to VDCs ([#948](https://github.com/vmware/terraform-provider-vcd/pull/948))
+* Resource and data source `vcd_catalog` and `vcd_subscribed_catalog` introduce new computed field `is_local` to specify
+  whether the catalog originated from the current org ([#949](https://github.com/vmware/terraform-provider-vcd/pull/949))
+* Improve usage of `org` field in `vcd_catalog` to accept sharing Org name for shared catalogs and improve error messages ([#949](https://github.com/vmware/terraform-provider-vcd/pull/949))
+
+### BUG FIXES
+* Fix a bug that caused `vcd_vm_group` data source to fail when the backing Provider VDC had multiple resource pools ([#948](https://github.com/vmware/terraform-provider-vcd/pull/948))
+* Fix issue #944 - shared catalog datasource not accessible to Org users ([#949](https://github.com/vmware/terraform-provider-vcd/pull/949))
+* Fix issue #672 - Update Org with invalid or extended LDAP settings ([#952](https://github.com/vmware/terraform-provider-vcd/pull/952), [#955](https://github.com/vmware/terraform-provider-vcd/pull/955))
```
Comment (lvirbalas) [inline CHANGELOG.md:13]: Just for consistency:

```suggestion
* Fix issue #672 - update Org with invalid or extended LDAP settings ([#952](https://github.com/vmware/terraform-provider-vcd/pull/952), [#955](https://github.com/vmware/terraform-provider-vcd/pull/955))
```

**Review by lvirbalas — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+
+### IMPROVEMENTS
+* Add `vdc_id` to data source `vcd_vm_placement_policy` to allow tenant users to fetch VM Placement Policies from
+  the ones assigned to VDCs ([#948](https://github.com/vmware/terraform-provider-vcd/pull/948))
+* Resource and data source `vcd_catalog` and `vcd_subscribed_catalog` introduce new computed field `is_local` to specify
+  whether the catalog originated from the current org ([#949](https://github.com/vmware/terraform-provider-vcd/pull/949))
+
+### BUG FIXES
+* Fix a bug that caused `vcd_vm_group` data source to fail when the backing Provider VDC had multiple resource pools ([#948](https://github.com/vmware/terraform-provider-vcd/pull/948))
+* Fix issue #944 shared catalog datasource not accessible to org users ([#949](https://github.com/vmware/terraform-provider-vcd/pull/949))
```
[the remark this answers is not in the document]
Comment (dataclouder) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 1 above)
Comment (dataclouder) [reply CHANGELOG.md]: done

**Pull request merged**